README.md

```diff
@@ -274,6 +274,11 @@ type Dispatcher = {
   waitFor: (dispatchTokens: Array<string>) => void;
 };
 ```
+## Redux Devtools Extension
+
+Using [Redux devtools extension](https://github.com/zalmoxisus/redux-devtools-extension) you can inspect the state of a store and see how the state changes between dispatches. The "Jump" feature should work but it is dependent on you using regular JS objects as the backing state.
+
+Using the `defineFactory` way of creating stores is highly recommended for this integration as you can define a name for your store and always for the state of the store to be inspected programmatically.
```

Comment:
> Using the `defineFactory` way of creating stores is highly recommended for this integration as you can define a name for your store and always for the state of the store to be inspected programmatically.

Does it work at all with `GeneralStore.define()` stores? Maybe just say "This only really works with `defineFactory()` stores"?

Reply:
It still works in the way that you can see each individual store and what actions it listened to but you don't see the state of the store.

Reply:
👌